File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/ClearUserIdAndToken.java
package com.project.scratchstudio.kith_andoid.SetInternalData;

import android.content.SharedPreferences;

public class ClearUserIdAndToken implements ISetInternalData {

    @Override
    public int set(SharedPreferences sp) {
        SharedPreferences.Editor ed;
        ed = sp.edit();
        ed.putInt("cur_user_id", -1);
        ed.putString("cur_user_token", "");
        ed.putString("cur_user_password", "");
        ed.apply();
        return 0;
    }

}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/SetInternalData/ClearUserIdAndToken.kt
package com.project.scratchstudio.kith_andoid.SetInternalData

import android.content.SharedPreferences

class ClearUserIdAndToken : ISetInternalData {

    override fun set(sp: SharedPreferences): Int {
        val ed: SharedPreferences.Editor = sp.edit()
        ed.putInt("cur_user_id", -1)
        ed.putString("user_token", "")
        ed.putString("cur_user_password", "")
        ed.apply()
        return 0
    }

}
